src/pages/_document.tsx

```diff
@@ -3,7 +3,9 @@ import { Html, Head, Main, NextScript } from "next/document";
 export default function Document() {
   return (
     <Html lang="en">
-      <Head />
+      <Head>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      </Head>
```

Comment:
Curious what this does

Reply:
It's for enabling up media queries in tailwind: https://tailwindcss.com/docs/responsive-design